src/rewriter.ts

```diff
@@ -92,10 +92,23 @@ export abstract class Rewriter {
     this.writeRange(pos, node.getEnd());
   }
 
-  // Write a span of the input file as expressed by absolute offsets.
-  // These offsets are found in attributes like node.getFullStart() and
-  // node.getEnd().
+  private skipToOffset = 0;
+
+  /**
+   * Skip emitting any code before the given offset. Used to avoid emitting @fileoverview comments
+   * twice.
+   */
+  skipUpToOffset(offset: number) {
```

Comment:
nit: I'd name the property and method the same. any particular reason it's not just a setter?